MIDIchlorians/MIDIchlorians/DataManager.swift

```diff
@@ -92,16 +96,20 @@ class DataManager {
         }
 
         do {
-            guard let session = loadSession(sessionName) else {
+            guard let session = loadExactSession(sessionName) else {
```

Comment:
This is a part of the removeSession method (difficult to see unless you expand) :)